.gitignore

```diff
@@ -58,3 +58,4 @@ rundeckapp/out
 rundeckapp/rundeck-runtime
 /rundeckapp/grails-app/assets/node_modules
 /rundeckapp/grails-app/assets/**/node_modules
+/rundeckapp/grails-app/conf/application-development.yml
```

Comment:
intended change/

Reply:
Yes, sometimes I use an application-development.yml to override defaults. Since these settings should be developer specific the file shouldn't be checked into source control.